.travis.yml

```diff
@@ -35,8 +35,7 @@ matrix:
       after_success:
       - ./src/gradlew -p src app::jacocoTestReport
       - echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin
-      - ./src/userCLI/scripts/build
-      - ./src/userCLI/scripts/push
+      - (cd src; ./src/userCLI/scripts/build; ./src/userCLI/scripts/push)
```

Comment:
These script calls were failing as not running in same folder as root gradle project...